File: src/hello_java/b_if/Ex_2884.java
package hello_java.b_if;

import java.time.LocalTime;
import java.util.Scanner;

/**
 * #2884 알람 시계
 *
 * 시간 뺄셈 문제
 */

public class Ex_2884 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);

        int hour = scanner.nextInt();
        int minute = scanner.nextInt();

        LocalTime time = LocalTime.of(hour,minute).minusMinutes(45);

        System.out.printf("%s %s", time.getHour(), time.getMinute());
    }
}

File: src/hello_kotlin/b_if/Ex_2884.kt
package hello_kotlin.b_if

import java.time.LocalTime
import java.util.*

/**
 * #2884 알람 시계
 *
 * 시간 뺄셈 문제
 */

fun main() = with(Scanner(System.`in`)){
    val hour = nextInt()
    val minute = nextInt()

    val time = LocalTime.of(hour, minute).minusMinutes(45)

    print("${time.hour} ${time.minute}")
}